client/js/components/procedure/StatementSegmentsList/StatementSegment.vue

```diff
@@ -463,18 +468,19 @@ export default {
         segmentId: this.segment.id,
         procedureId: this.procedureId
       },
-      allComponentsLoaded: false,
-      asyncComponents: [],
-      showWorkflowActions: false,
-      selectedAssignee: {},
+      allAddonsLoaded: false,
       claimLoading: false,
       currentUserName: this.currentUserFirstName + ' ' + this.currentUserLastName,
+      demosplanUi: shallowRef(demosplanUi),
       isCollapsed: !(this.segment.relationships?.assignee?.data && this.segment.relationships.assignee.data.id === this.currentUserId),
       isEditing: false,
       isFullscreen: false,
       isHover: false,
+      loadedAddons: [],
```

Comment:
I would rename this to `recommendationModalAddons` or something similar.

Reply:
done here https://github.com/demos-europe/demosplan-core/pull/4571/commits/cf99d1f172b79954b2935d2fc0e91298389e3995